tests/nested/core20/basic/task.yaml

```diff
@@ -27,4 +30,4 @@ execute: |
     execute_remote "sudo snap remove test-snapd-sh" || true
 
     echo "Ensure 'snap list' works and test-snapd-sh snap is removed"
-    execute_remote "snap list" | NOMATCH test-snapd-sh
+    execute_remote "snap list" | not MATCH test-snapd-sh
```

Comment:
NOMATCH and not MATCH are sadly different. Perhaps we should just check this instead: `not execute_remote "snap list test-snapd-sh"`

Reply:
Nice idea with `snap list test-snapd-sh`

Reply:
sure sounds good to me